source/developer_manual/design/dsync.rst

```diff
@@ -0,0 +1,125 @@
+.. _doveadm_dsync:
+
+============
+Dsync Design
+============
+
+FIXME: This describes the design for v2.0/v2.1. The v2.2 design is
```

Comment:
maybe this should be fixed?

Reply:
I don't know much about dsync can anybody help me out updating that page?

Reply:
probably should write new jira ticket to fix this.